nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -150,36 +157,38 @@ def handoff_items_phase(self):
         rospy.loginfo("Nightingale Mission Planner starting to hand items")
         # Arrived at patient's bedside
 
+        # pose estimation
```

Comment:
were leaving the pose estimation commented out for now to make arm testing easer